src/components/multiSelect/MultiSelectBottomSheet.tsx

```diff
@@ -0,0 +1,232 @@
+import GorhomBottomSheet, { BottomSheetProps } from '@gorhom/bottom-sheet'
+import React, { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import FastImage from 'react-native-fast-image'
+import { ScrollView } from 'react-native-gesture-handler'
+import BottomSheetBase from 'src/components/BottomSheetBase'
+import BottomSheetScrollView from 'src/components/BottomSheetScrollView'
+import Touchable from 'src/components/Touchable'
+import Checkmark from 'src/icons/Checkmark'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+const OPTION_HEIGHT = 60
+const MAX_OPTIONS_IN_VIEW = 5
+
+interface MultiSelectBottomSheetProps<T extends string> {
+  forwardedRef: React.RefObject<GorhomBottomSheet>
+  onChange?: BottomSheetProps['onChange']
+  onClose?: () => void
+  onOpen?: () => void
+  handleComponent?: BottomSheetProps['handleComponent']
+  options: Option[]
+  selectedOptions: Option[]
+  setSelectedOptions: (selectedOptions: Option[]) => void
```

Comment:
these can just be `string[]` or `T[]` and just include the id right? Is there a reason to pass around all the fields in Option? Using Sets are also an option if we wanted to avoid duplicates